doc/config.md

```diff
@@ -608,6 +608,12 @@ Works only with UDNS and c-ares backends (`configure` option `--with-udns` or `-
 
 Default: 0.0 (disabled)
 
+### dns_close_previous
```

Comment:
bikeshed: I think `dns_close_previous` is not a great name for this option. I think `removed` or `missing` is a better suffix.
```suggestion
### dns_close_removed
```